R/arx_forecaster.R

```diff
@@ -34,7 +34,7 @@ arx_forecaster <- function(x, y, key_vars, time_value,
   }
 
   dat <- create_lags_and_leads(x, y, lags, ahead, time_value, keys)
-  if (intercept) dat$x0 <- 1
```

Comment:
OOC, where was this intercept coming from before? I don't see it passed in or defined above

Reply:
Based on last week's meeting, I think that is obsolete.